### PR TITLE
set page titles for non project page

### PR DESCRIPTION
**Pull request by Lennerblom**

### CODE REVIEW

File: common/components/controllers/SignedUpController.jsx [hunk 1]
```diff
@@ -12,6 +13,11 @@ class SignedUpController extends React.Component<{||}> {
   render(): React$Node {
     // TODO: Give better indication that email has been sent when user clicks resend link
     return (
+      <React.Fragment>
+      <Headers
+      title="Sign Up| DemocracyLab"
```
Comment (marlonkeating) [inline common/components/controllers/SignedUpController.jsx]: Actually this should go in SignUpController.jsx.  This file is for the page that appears after a user signs up.

Comment (Lennerblom) [reply common/components/controllers/SignedUpController.jsx]: I added a new commit to address the requested changes

File: common/components/controllers/PartnerWithUsController.jsx [hunk 2]
```diff
@@ -11,6 +11,11 @@ class PartnerWithUsController extends React.Component<{||}> {
 
   render(): React$Node {
     return (
+      <React.Fragment>
+      <Headers
```
Comment (marlonkeating) [inline common/components/controllers/PartnerWithUsController.jsx]: Import Headers

File: common/components/controllers/SignUpController.jsx [hunk 3]
```diff
@@ -82,6 +82,11 @@ class SignUpController extends React.Component<Props, State> {
 
   render(): React$Node {
     return (
+      <React.Fragment>
+      <Headers
```
Comment (marlonkeating) [inline common/components/controllers/SignUpController.jsx]: Import Headers

File: common/components/controllers/SignUpController.jsx [hunk 4]
```diff
@@ -82,6 +83,11 @@ class SignUpController extends React.Component<Props, State> {
...
+      title="Sign Up| DemocracyLab"
```
Comment (marlonkeating) [inline common/components/controllers/SignUpController.jsx]:
```suggestion
      title="Sign Up | DemocracyLab"
```

**Pull request merged**